samples/sensor/tmp116/src/main.c

```diff
@@ -14,13 +14,31 @@ void main(void)
 {
 	const struct device *dev;
 	struct sensor_value temp_value;
+
+	/* offset to be added to the temperature
+	 * only supported by TMP117
+	 */
+	struct sensor_value offset_value;
 	int ret;
 
 	dev = device_get_binding(DT_LABEL(DT_INST(0, ti_tmp116)));
 	__ASSERT(dev != NULL, "Failed to get TMP116 device binding");
 
 	printk("Device %s - %p is ready\n", dev->name, dev);
 
+	/*
+	 * if an offset of 2.5 oC is to be added,
+	 * set val1 = 2 and val2 = 500000.
+	 * See struct sensor_value documentation for more details.
+	 */
+	offset_value.val1 = 0;
+	offset_value.val2 = 300000;
```

Comment:
It will be confusing for users to have 0.3 offset. I think 0.0 offset (as it was before) makes more sense for this sample.

Reply:
ohh.. sorry, I forgot to change that.. I was testing it so had it at 0.3

Reply:
changed it.